mlrun/model_monitoring/db/tsdb/timescaledb/timescaledb_connector.py

```diff
@@ -0,0 +1,123 @@
+# Alternative approach using composition instead of inheritance
+
+from typing import Optional
+
+import mlrun.common.schemas.model_monitoring as mm_schemas
+from mlrun.datastore.datastore_profile import DatastoreProfileTimescaleDB
+from mlrun.model_monitoring.db import TSDBConnector
+from mlrun.model_monitoring.db.tsdb.preaggregate import PreAggregateConfig
+from mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_connection import (
+    TimescaleDBConnection,
+)
+from mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_operations import (
+    TimescaleDBOperationsHandler,
+)
+from mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_query_handler import (
+    TimescaleDBQueryHandler,
+)
+from mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_stream import (
+    TimescaleDBStreamHandler,
+)
+
+
+class TimescaleDBConnector(TSDBConnector):
+    """
+    Complete TimescaleDB TSDB connector using composition
+    """
+
+    type: str = mm_schemas.TSDBTarget.TimescaleDB
```

Comment:
I guess it should be
```suggestion
    type: ClassVar[str] = mm_schemas.TSDBTarget.TimescaleDB
```
to avoid a static check warning.